src/main.ts

```diff
@@ -7,12 +7,13 @@ import { handlePullRequestMessage } from './libs/pr';
 import * as pulumiCli from './libs/pulumi-cli';
 import { invariant } from './libs/utils';
 
+const pulumiVersion = '^3';
+
 const main = async () => {
   const config = await makeConfig();
   core.debug('Configuration is loaded');
 
-  invariant(pulumiCli.isAvailable(), 'Pulumi CLI is not available.');
-  core.debug('Pulumi CLI is available');
+  await pulumiCli.downloadCli(pulumiVersion);
 
```

Comment:
should we add a debug message about what version is now available?

Reply:
We actually output the range in the `downloadCli` function, along with matched version.

Reply:
wonderful!!